contrib/devtools/Makefile

```diff
@@ -96,26 +96,13 @@ protoc-gen-gocosmos:
 	@echo "Installing protoc-gen-gocosmos..."
 	@go install github.com/regen-network/cosmos-proto/protoc-gen-gocosmos
 
-buf: protoc-gen-buf-check-breaking protoc-gen-buf-check-lint
+buf:
 	@echo "Installing buf..."
 	@curl -sSL \
     "https://github.com/bufbuild/buf/releases/download/v${BUF_VERSION}/buf-${UNAME_S}-${UNAME_M}" \
     -o "${BIN}/buf" && \
```

Comment:
The BIN variable should be set to `$(go env GOPATH)/bin` as in some systems you need root privileges to install files in `/usr/local/bin`